jellyfin.tf

```diff
@@ -11,6 +11,10 @@ terraform {
   }
 }
 
+locals {
+  server_name = var.HOSTED_ZONE_ID == "" ? aws_lb.jellyfin_alb.dns_name : trimsuffix(data.aws_route53_zone.jellyfin_domain.0.name,".")
```

Comment:
Great.

Reply:
I replaced this your older version of alb name for you comment about the nginx conf